attack.go

```diff
@@ -186,7 +186,7 @@ func (h headers) Set(value string) error {
 	if key == "" || val == "" {
 		return fmt.Errorf("header '%s' has a wrong format", value)
 	}
-	h.Add(key, val)
+	h.Header[key] = []string{val}
```

Comment:
I thought this was a bug since your method is called `Set`, not `Add`.
vegeta's `headers.Set` also accepts a full header `key:value` pair, so it makes sense to overwrite the complete combination. Is it currently being used to add multiple values to the same key?